blivet/arch.py

```diff
@@ -182,27 +182,6 @@ def is_aarch64():
     return os.uname()[4] == 'aarch64'
 
 
-def get_arm_machine():
-    """
-    :return: The ARM processor variety type, or None if not ARM.
-    :rtype: string
-
-    """
-    if not is_arm():
-        return None
-
-    if flags.arm_platform:
```

Comment:
Please also remove the (now unused) flags import to make pylint happy.

Reply:
Will do, update coming in shortly, I'll also remove the "self.arm_platform = None" declare from flags.py too